app/templates/admin/add_agency.html

```diff
@@ -0,0 +1,33 @@
+{% extends 'layouts/base.html' %}
+{% import 'macros/form_macros.html' as f %}
+
+{% block content %}
+    <div class="ui grid container">
+        <div class="eight wide computer sixteen wide mobile centered column">
+            <h2 class="ui dividing header">Create an agency</h2>
+
+            {% set flashes = {
```

Comment:
not particularly pressing/something to consider - should we refactor the flashes stuff if we're using repeated code? 
`git grep -i "set flashes"` to see everywhere it's used, which is at least 4 times in our code base